spec/Section 3 -- Type System.md

```diff
@@ -232,6 +232,152 @@ Schema extensions have the potential to be invalid if incorrectly defined.
 2. Any non-repeatable directives provided must not already apply to the
    original Schema.
 
+### Schema Coordinates
+
+Schema Coordinates are human readable strings that uniquely identify an element defined in a GraphQL Schema.
+
+**Definition**
+
+SchemaCoordinate:
+  - Name
+  - Name . Name
+  - Name . Name ( Name : )
+  - @ Name
+  - @ Name ( Name : )
+
+**Semantics**
+
+SchemaCoordinate: Name
+  1. Let {typeName} be the value of the first {Name}.
+  2. Return the type in the schema named {typeName}.
+
+SchemaCoordinate: Name . Name
+  1. Let {typeName} be the value of the first {Name}.
+  2. Let {type} be the type in the schema named {typeName}.
+  3. If {type} is an Enum type:
+    1. Let {enumName} be the value of the second {Name}.
+    2. Return the enum value of {type} named {enumName}.
+  4. Otherwise if {type} is an Input Object type:
+    1. Let {inputFieldName} be the value of the second {Name}.
+    2. Return the input field of {type} named {inputFieldName}.
+  5. Otherwise {type} must be an Object or Interface type:
+    1. Let {fieldName} be the value of the second {Name}.
+    2. Return the field of {type} named {fieldName}.
```

Comment:
I think making the assertion explicit is more in keeping with the rest of the algorithm:

```suggestion
  5. Otherwise:
    1. Assert: {type} must be an Object or Interface type.
    2. Let {fieldName} be the value of the second {Name}.
    3. Return the field of {type} named {fieldName}.
```